src/rtcModule/webrtc.cpp

```diff
@@ -173,6 +173,40 @@ bool RtcModule::selectDevice(const std::string& devname,
     }
 }
 
+void RtcModule::updateConstraints(RtcModule::Resolution resolution)
+{
+    switch (resolution)
+    {
+        case Resolution::hd:
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMaxHeight, 1080);
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMinHeight, 576);
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMaxWidth, 1920);
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMinWidth, 1024);
+            break;
+
+        case Resolution::low:
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMaxHeight, 288);
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMinHeight, 240);
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMaxWidth, 352);
+            mMediaConstraints.SetMandatory(webrtc::MediaConstraintsInterface::kMinWidth, 320);
+            break;
+
+        case Resolution::vga:
```

Comment:
We could use fall-through for this case, since it's the same than for `default`

Reply:
Solved it